Improved UI and added characters

aaronloh16 opened the pull request:
![Screenshot 2025-03-16 at 7 57 04 PM](https://github.com/user-attachments/assets/cf438e01-c7d6-4bca-8487-e8aa6e6bf26e)
![Screenshot 2025-03-16 at 7 57 16 PM](https://github.com/user-attachments/assets/3930dfd0-f41a-4bed-b212-77804564deed)

aaronloh16 committed:
Improved UI and added characters

aaronloh16 committed:
Added images

owengretzinger commented on frontend/app/story/page.tsx:
```diff
@@ -8,48 +9,90 @@ import { SuspectCard } from '@/components/suspects/SuspectCard';
+							On a stormy night at the TechVision headquarters, CEO and founder
+							Thomas Richardson was found dead in his executive office on the
+							top floor. The cause of death: a single gunshot wound to the
+							chest. The murder weapon, his own custom engraved handgun, is
+							missing.
```
How do we know the murder weapon was his own custom engraved handgun if it's missing

aaronloh16 replied:
Haha let me think about how to make the story actually good, that will be my next task, that was mostly AI generated

owengretzinger reviewed (approved):
Great stuff!

aaronloh16 committed:
feat(database): implement database-driven architecture

- Set up Drizzle ORM with Neon PostgreSQL
- Create schema with suspects, contexts, secrets, and story tables
- Generate and apply database migrations
- Create seeding script for initial data
- Develop API endpoints for suspects and story data
- Update frontend components to use data from APIs
- Add loading and error states for better UX

aaronloh16 committed:
update README

aaronloh16 committed:
Delete .cursor/rules directory

aaronloh16 committed:
Delete memory-bank directory
